doc/api/process.md

```diff
@@ -2158,6 +2158,14 @@ a [Writable][] stream.
 `process.stderr` differs from other Node.js streams in important ways. See
 [note on process I/O][] for more information.
 
+### `process.stderr.fd`
+
+* {number}
+
+This property refers to the value of underlying file descriptor of
+`process.stderr`. The value is fixed at `2`. In `worker_thread`s,
```

Comment:
```suggestion
`process.stderr`. The value is fixed at `2`. In [`Worker`][] threads,
```

ditto elsewhere. This also matches the usage in the rest of the documentation.

Reply:
@addaleax, thanks. Fixed it.